Editor/Modules/TextureModule.cs

```diff
@@ -0,0 +1,114 @@
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.Name), format = PropertyFormat.String, name = "Name", longName = "Show Texture Name" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.Format), format = PropertyFormat.String, name = "Format", longName = "Show Texture Format" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.Shape), format = PropertyFormat.String, name = "Shape", longName = "Show Texture Shape" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.ReadWrite), format = PropertyFormat.String, name = "Writeable", longName = "Show Texture ReadWrite" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.StreamingMipMaps), format = PropertyFormat.String, name = "StreamingMipMaps", longName = "Show Texture StreamingMipMaps" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.MinMipMapLevel), format = PropertyFormat.String, name = "MinMipMapLevel", longName = "Show Texture MinMipMapLevel" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.Resolution), format = PropertyFormat.String, name = "Resolution", longName = "Show Texture Resolution" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.SizeOnDisk), format = PropertyFormat.String, name = "SizeOnDisk", longName = "Show Texture SizeOnDisk" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.Path), format = PropertyFormat.String, name = "Location", longName = "Show Texture Location Path" }
+                //  new PropertyDefinition { type = PropertyType.Area, name = "Texture Recommendations", longName = "Recommendations for Optimizations for the Textures in the project." }
+                // above line makes stuff puke while the module is half-implemented
+            }
+        };
+
+        public override IEnumerable<IssueLayout> GetLayouts()
+        {
+            yield return k_IssueLayout;
+        }
+
+        public override void Audit(ProjectAuditorParams projectAuditorParams, IProgress progress = null)
+        {
+            // Implement your analysis here
+            var allTextures = AssetDatabase.FindAssets("t: Texture, a:assets");  // t: filter is for type of asset returned and a: filter is for location to search (only this location)
+            // Debug.Log("Locations Searched: " + String.Join(" & ", new List<string>(searchTheseFolders).ConvertAll(i => i.ToString())));
+            // Create an issue
+            var issues = new List<ProjectIssue>();
+
+
+            // Grab the texture's path/location and add it as an issue.
+            foreach (string aTexture in allTextures)    // ACTUALLY for each texture referenced....
+            {
```

Comment:
I think this is a good use-case for the _progress_ object

Reply:
In-Progress. Not Pushed yet.

Reply:
Progress added.